src/hooks/usePreloader.tsx

```diff
@@ -0,0 +1,17 @@
+import { useEffect } from 'react';
+
+const usePreloader = (isLoaded: boolean) => {
+  const checkData = (): void => {
+    const preloader = document.querySelector('.preloader');
+
+    if(isLoaded && preloader) {
+      preloader.remove();
+    }
+  }
+
+  useEffect(() => {
+    checkData();
```

Comment:
maybe add a deps array with isLoaded? 
Also we need to check data only if isLoaded = true? 
The we can add check to useEffect, and rename function to removeLoader and not use isLoaded inside that function